govstack_api/building_blocks/bb_digital_registries/registries/base_registry.py

```diff
@@ -0,0 +1,12 @@
+class BaseRegistry:
+    def get_record(self):
+        pass
+
+    def update_record(self):
+        pass
+
+    def delete_record(self):
+        pass
+
+    def check_if_record_exists(self):
+        pass
```

Comment:
I think you can use typehint `Protocol` inheritance in here to create implicit interface (it will be more pythinic than using abstract classes). I wouldn't call this one 'BaseRegistry' as there is no base funcionality provided.

Reply:
Ok, done